TheaterSchedule/Controllers/DBHelperController.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Text;
+using System.Collections.Generic;
+using System.Linq;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+
+using TheaterSchedule.Util;
+using TheaterSchedule.DAL.Contexts;
+using TheaterSchedule.DAL.Repositories;
+using TheaterSchedule.DAL.Interfaces;
+using TheaterSchedule.DAL.Entities;
+
+namespace TheaterSchedule.Controllers
+{
+    [Route("api/[controller]")]
+    public class DBHelperController : Controller
+    {
+        private readonly ITheaterScheduleUnitOfWork uow;
+        private readonly IHostingEnvironment appEnvironment;
+
+        public DBHelperController(ITheaterScheduleUnitOfWork _uow, IHostingEnvironment _appEnvironment)
+        {
+            uow = _uow;
+            appEnvironment = _appEnvironment;
+        }
+
+        [HttpGet]
+        public string Get()
+        {
+            return "To fill DB with default data make request \"/api/DBHelper/UseDefaultData\" with POST method or visit page \"/FillDB.html\"";
+        }
+
+
+        [HttpPost("UseDefaultData")]
+        public string Post()
+        {
+            string path = Path.Combine(appEnvironment.ContentRootPath, "static/media/");
+
+            uow.Performances.Create(new Performance { MainImage = ImageHelper.ImageToByteArray(path + "dva-levy.png"), Duration = 100, MinPrice = 40, MaxPrice = 120, MinimumAge = 2 });
+            uow.Performances.Create(new Performance { MainImage = ImageHelper.ImageToByteArray(path + "kit-u-chobotjah.png"), Duration = 70, MinPrice = 35, MaxPrice = 80, MinimumAge = 3 });
+
+            //...filling_other_tables...
+            uow.Save();
+
+            //ImageHelper.ByteArrayToImage(((Performance)uow.Performances.GetAll().First()).MainImage).Save("D:\\test.png");
```

Comment:
don't commit commented code
or add comment with TODO and ticket number what to implement